### PR TITLE
add a workflow script so one doesn't have to do it manually all the time

### PR DESCRIPTION
**Pull request by dushyantbehl**
This commit adds a script to run contributing workflow on a new project and ingress firewall generated using the script.

### CODE REVIEW

**Review by palanik1 — CHANGES_REQUESTED** [review]
@dushyantbehl Thank you! The script looks good.Can you split this PR to separate out the script from the ingress-node-firewall annotations. Also, can you update the contributing_workflow.md to include how to run your script.

**Comment by palanik1** [comment]
@dushyantbehl also...tag your PR to an existing issue/or create one if it does not exist.

**Comment by dushyantbehl** [comment]
Ah...they got squashed with my signing...let me separate them out

File: contribution_workflow.md [hunk 1]
```diff
@@ -1,10 +1,65 @@
...
+
+* To create necessary directories user can run the below command
+
+`./scripts/execute_workflow.sh create_dirs ingress-node-firewall-master https://github.com/openshift/ingress-node-firewall.git`
+
+When run in a shell the output will look like below,
+
+```
+$ ./scripts/execute_workflow.sh create_dirs ingress-node-firewall-master https://github.com/openshift/ingress-node-firewall.git
+Run the below command and keep the bpf directory only, deleting others
```
Comment (palanik1) [inline contribution_workflow.md:43]: I was suggesting splitting out the command from the output like
```
$ ./scripts/execute_workflow.sh create_dirs ingress-node-firewall-master https://github.com/openshift/ingress-node-firewall.git
```


Run the below command and keep the bpf directory only, deleting others

```
git clone --depth=1 https://github.com/openshift/ingress-node-firewall.git projects/ingress-node-firewall-master/original_source && rm -rf projects/ingress-node-firewall-master/original_source/.git && cd projects/ingress-node-firewall-master/original_source
```

**Review by palanik1 — APPROVED** [review]
lgtm

**Pull request merged**